### ISSUE TITLE
Geometric Deep Learning

### ISSUE THREAD

**Issue by CaiYitao** [FluxML/GeometricFlux.jl#284]
Hi Yuehua,
I was wondering if this package is based on the  [geometric deep learning](https://geometricdeeplearning.com/)?  Can this package do the 5G(Grids,Graphs,Groups,Geodesics&Gauges) thing?

Thank you!
Yitao

**Comment by yuehhua** [comment]
Currently, this package is towards the goal of [geometric deep learning](https://geometricdeeplearning.com/) by Michael Bronstein. However, most of the things in this package is not ready yet. For the 5G, Grids and Graphs are available.

It's welcome to contribute models from the paper [a geometric understanding of deep learning](https://www.sciencedirect.com/science/article/pii/S2095809919302279) into this package.

**Comment by CaiYitao** [comment]
That sounds very exciting!!!  
Right now, my skill is not up to contribute to the package, I will try to practice Julia more often, when I feel I am ready I will, till then perhaps also gonna need your help: )

**Comment by yuehhua** [comment]
@CaiYitao Do you refer [this paper](https://openreview.net/forum?id=o1O5nc48rn)?

**Comment by CaiYitao** [comment]
yeah, I am aware of this paper, and I like the idea of this paper, but haven't dug deep into it yet, I was thinking of applying [a geometric understanding of deep learning](https://www.sciencedirect.com/science/article/pii/S2095809919302279) with geometric deep learning /GNN

**Comment by yuehhua** [comment]
I just check the [paper](https://www.sciencedirect.com/science/article/pii/S2095809919302279). I seems to analyze and establish the relationship between optimal transportation (OT) and generative adversarial networks (GANs). In this paper, it proposed a novel generative model AE-OT model, and it uses an autoencoder (AE) for manifold learning and OT map for probability distribution transformation. I am sure it is a kind of deep learning model, but should not under the umbrella of geometric deep learning. Although there are some relationship between geometric deep learning and OT, the relationship remains not mentioned in this paper.

**Comment by yuehhua** [comment]
ConfGF to be https://github.com/DeepGraphLearning/ConfGF? And SEGNN to be https://github.com/EnyanDai/SEGNN?

**Comment by CaiYitao** [comment]
Yeah, ConfGF is that one, but SEGNN is this [SEGNN](https://github.com/robdhess/steerable-e3-gnn)
I think their method is pretty general not constrained by what presented in the paper AE-OT model and plausible cause this method developed together with Shingtung Yau(丘成桐）

**Comment by yuehhua** [comment]
I think it's OK they didn't aware of geometric deep learning developed by Bronstein et.al. I didn't go deeper in this paper but I think it is surely general enough to extend to geometric deep learning. However, this package should collect the models under the umbrella of geometric deep learning, at least. There might be some work to be done and we can have models in this package.

**Comment by CaiYitao** [comment]
I am very glad and grateful that you are developing these amazing packages,  I am planning to use this package to write my master thesis, and I hope in the meantime I will be proficient enough to contribute : ).

**Comment by CaiYitao** [comment]
The code for applying OT,GNN to protein analysis, was out [here](https://github.com/octavian-ganea/equidock_public)
